extension/cart/handleCartNotFound.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {Error} error
+ * @param {SDKContext} context
+ */
+module.exports = async (error, context) => {
+  if (!error) {
+    return { cartNotFound: false }
+  }
+
+  if (error.code && error.code === 'ENOTFOUND') {
+    const storage = !context.meta.userId ? 'device' : 'user'
+    await context.storage[storage].del('cartId')
+
+    return { cartNotFound: true }
+  }
+
+  throw error
```

Comment:
if u throw an error here, does the static pipeline stuff still work? I thought we were not throwing this error at user's face? Are we?

Reply:
Only errors that are not ENOTFOUND errors are again thrown, because we are not handling them in here.